### ISSUE TITLE
[Video / Issue]: 

### ISSUE THREAD

**Issue by bspmail** [archetyped/slb-addons#19]
## Description of issue

I have simple lightbox installed and I purchased access to the video plugin beta but it doesn't seem to work.  I have the latest wordpress and latest simple lightbox installed but when I put a youtube link in a post it just plays in the post as normal, there is not a video in a lightbox.  Any suggestions??

Thanks,
Bryan

## Details

* URL of page exhibiting the issue: 
* Web Browser(s) that exhibit the issue: 
* Simple Lightbox Version: 
* Active theme: 
* Other plugins installed:

**Comment by archetyped** [comment]
Hi, please fill in the details of the issue you are experiencing and I would be glad to take a closer look.

**Comment by bspmail** [comment]
Thanks, I put the info in my original post but here are the details, I have
latest Wordpress and latest simple lightbox plugin installed.  I purchased
and installed the video beta plugin which says it works with youtube vids.
I created some sample posts with youtube videos and they play normally in
the post, i.e. not in a light box.  I followed instructions for
installation/activation of the plugin but it doesn't seem to be working.

Any ideas?

Thanks,
Bryan
